test/e2e/dra/dra.go

```diff
@@ -773,6 +775,54 @@ var _ = ginkgo.Describe("[sig-node] DRA [Feature:DynamicResourceAllocation]", fu
 		multipleDriversContext("using only drapbv1alpha3", false, true)
 		multipleDriversContext("using both drapbv1alpha2 and drapbv1alpha3", true, true)
 	})
+
+	// This test covers aspects of non graceful node shutdown by DRA controller
+	// More details about this can be found in the KEP:
+	// https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2268-non-graceful-shutdown
+	ginkgo.Context("cluster", func() {
+		nodes := NewNodes(f, 2, 8)
+		driver := NewDriver(f, nodes, networkResources)
+		b := newBuilder(f, driver)
+
+		// NOTE: this test depends on kind. It will only work with kind cluster as it shuts down one of the
+		// nodes by running `docker stop <node name>`, which is very kind-specific.
```

Comment:
this should not be inside the e2e framework, we should not start mixing deployment and testing

Reply:
oh, mybad, sorry, is an independent folder, so sorry